fullscreen.bs

```diff
@@ -79,34 +79,33 @@ its <a>node document</a>'s <a>top layer</a>.
 <p>To <dfn>fully exit fullscreen</dfn> a <a for=/>document</a> <var>document</var>, run these steps:
 
 <ol>
- <li><p>If <var>document</var>'s <a>fullscreen element</a> is null, terminate these steps.
+ <li><p>Let <var>topLevelDoc</var> be <var>document</var>'s <a>top-level browsing context</a>'s
+ <a>active document</a>.
 
- <li><p><a lt="unfullscreen an element">Unfullscreen elements</a> whose <a>fullscreen flag</a> is
- set, within <var>document</var>'s <a>top layer</a>, except for <var>document</var>'s
- <a>fullscreen element</a>.
+ <li><p>If <var>topLevelDoc</var>'s <a>fullscreen element</a> is null, terminate these steps.
```

Comment:
How should this interact with the changes in https://github.com/whatwg/fullscreen/pull/102? The "fullscreen element is null" condition should be impossible if top layer is modified *after* the synchronous part of this algorithm has run.

Reply:
The condition is possible because there are other callsites to this algorithm, for example, unloading document, and probably for the escaping mechanism. If we don't check here, we would need to guard all callsites with this condition, and we probably still want to assert it in this algorithm to ensure everything runs as expected.